src/open3DTool/drawingUtils.py

```diff
@@ -0,0 +1,23 @@
+from src.open3DTool.visualizer import Visualizer
+
+import open3d as o3d
+import numpy as np
+
+
+def draw_and_pick_points_function(
```

Comment:
Does it make sense to create a function with absolutely the same signature as `Visualizer` constructor?